src/screens/Learn/Faq/Faq.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+  max-width: 800px;
+  width: 100%;
+  padding: 0 1rem;
+  margin: 4rem auto;
+`;
+
+export const PageHeader = styled.h1`
+  margin-bottom: 1.75rem;
+`;
+
+export const SectionHeader = styled.h2`
+  font-weight: 900;
+  font-size: 1.5rem;
+  line-height: 1.25;
+  margin: 1.25rem 0;
+`;
```

Comment:
Maybe we should make these styles more generic, so we can reuse them in other pages on Learn (we can do that on a follow-up PR)

Reply:
added a todo in the notes above